pytorch_lightning/plugins/training_type/tpu_spawn.py

```diff
@@ -121,8 +121,20 @@ def pre_dispatch(self):
         if self.debug:
             os.environ["PT_XLA_DEBUG"] = str(1)
 
-    def setup(self) -> None:
+    def setup(self, trainer: "pl.Trainer") -> None:
+        # Revisit strategy inheritance
```

Comment:
since we have an issue for this now (#10685), we can remove the comment (assuming you meant this)

Reply:
If kept, it's missing a `TODO:`. Also don't forget about the occurrence in `pytorch_lightning/plugins/training_type/single_tpu.py`